.github/workflows/python_unit_test.yaml

```diff
@@ -28,22 +39,28 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
-    
+
     steps:
       - name: Check out code
         uses: actions/checkout@v3
       - name: Set up python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.6.7"
+          python-version: ${{ inputs.python-version }}
       - name: Install poetry
         run: |
-            pip --version
-            pip install poetry
-            poetry install -vv
+          pip --version
+          pip install poetry
+          poetry install -vv
       - name: Service specific setup
         run: |
-            curl -sSL "${{inputs.setup-script-location}}" | bash
+          curl -sSL "${{inputs.setup-script-location}}" | bash
```

Comment:
i see that `setup-script-location` is optional now, but there is no logic to skip this step if it's not provided. I assume this step would fail?

Reply:
oh it looks like you just forgot to set the default to `default_ci_setup.sh`